scio-google-cloud-platform/src/main/java/com/spotify/scio/bigtable/BigtableUtil.java

```diff
@@ -62,15 +68,24 @@ private BigtableUtil() {}
    * end to lower costs yet still get high throughput during bulk ingests/dumps.
    *
    * @param bigtableOptions Bigtable Options
-   * @param numberOfNodes New number of nodes in the cluster
-   * @param sleepDuration How long to sleep after updating the number of nodes. Google recommends at
-   *     least 20 minutes before the new nodes are fully functional
-   * @throws IOException If setting up channel pool fails
+   * @param numberOfNodes   New number of nodes in the cluster
+   * @param sleepDuration   How long to sleep after updating the number of nodes. Google recommends at
+   *                        least 20 minutes before the new nodes are fully functional
+   * @throws IOException          If setting up channel pool fails
    * @throws InterruptedException If sleep fails
    */
   public static void updateNumberOfBigtableNodes(
-      final BigtableOptions bigtableOptions, final int numberOfNodes, final Duration sleepDuration)
-      throws IOException, InterruptedException {
+      final BigtableOptions bigtableOptions,
+      final int numberOfNodes,
+      final Duration sleepDuration,
+      final Optional<Set<String>> clusterNamesOpt) throws IOException, InterruptedException {
```

Comment:
Instead of adding and `Optional` here we can simply keep as a `Set` and then add 

```java
  public static void updateNumberOfBigtableNodes(
       final BigtableOptions bigtableOptions, final int numberOfNodes, final Duration sleepDuration) {

List<Cluster> clusters = clustersResponse
           .getClustersList();

return updateNumberOfBigtableNodes(bigtableOptions, numberOfNodes, sleepDuration, clusters);
}

  public static void updateNumberOfBigtableNodes(
       final BigtableOptions bigtableOptions, final int numberOfNodes, final Duration sleepDuration, Set<String> clusterNamesOpt) {

List<Cluster> clusters =  clusters = clusters
             .stream()
             .filter(cluster -> clustersToScale.contains(shorterName(cluster.getName())))
             .collect(Collectors.toList());

return updateNumberOfBigtableNodes(bigtableOptions, numberOfNodes, sleepDuration, clusters);
}
```